examples/customer-reviews/src/app/api/fulfill/route.ts

```diff
@@ -0,0 +1,32 @@
+import { fulfillController } from "@/app/api/fulfill/fulfill.controller";
+import { type FulfillRequest, type FulfillResponse, fulfillRequestSchema } from "@/app/api/fulfill/fulfill.validation";
+import { getLogger } from "@local/utils";
+import { type NextRequest, NextResponse } from "next/server";
+
+export const maxDuration = 300;
+
+/**
+ * Continues with the task after the user has completed sign-in.
+ */
+export async function POST(request: NextRequest): Promise<NextResponse<FulfillResponse>> {
+  const log = getLogger().withPrefix("[api/fulfill]");
+
+  const data = (await request.json()) as FulfillRequest;
+  log.warn("=== Fulfill Request");
+  console.log(data);
```

Comment:
i wouldn't log the data because the api key will be logged and we dont want that

Reply:
I'm deleting all these logs because those were used only for debugging